### ISSUE TITLE
Error, pairs.K25.stats is empty. Be sure to check your fastq reads and ensure that the read names are identical except for the /1 or /2 designation. 

### ISSUE THREAD

**Issue by niehu2018** [trinityrnaseq/trinityrnaseq#524]
It is the same problem as #514.

Log file:
Left read files: $VAR1 = [
          'test_R1.fq'
        ];
Right read files: $VAR1 = [
          'test_R2.fq'
        ];
Trinity version: Trinity-v2.4.0
-ERROR: couldn't run the network check to confirm latest Trinity software version.

Saturday, July 21, 2018: 14:20:27	CMD: java -Xmx64m -XX:ParallelGCThreads=2 -jar /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/support_scripts/ExitTester.jar 0
Saturday, July 21, 2018: 14:20:27	CMD: java -Xmx64m -XX:ParallelGCThreads=2 -jar /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/support_scripts/ExitTester.jar 1
Saturday, July 21, 2018: 14:20:27	CMD: mkdir -p /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/trinity_test2
Saturday, July 21, 2018: 14:20:27	CMD: mkdir -p /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/trinity_test2/chrysalis

> ----------------------------------------------------------------------------------
> -------------- Trinity Phase 1: Clustering of RNA-Seq Reads  ---------------------
> ----------------------------------------------------------------------------------
> 
> ---------------------------------------------------------------
> ------------ In silico Read Normalization ---------------------
> -- (Removing Excess Reads Beyond 50 Coverage --
> ---------------------------------------------------------------
> 
> # running normalization on reads: $VAR1 = [
>           [
>             '/public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R1.fq'
>           ],
>           [
>             '/public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R2.fq'
>           ]
>         ];
> 
> 
> Saturday, July 21, 2018: 14:20:27	CMD: /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/insilico_read_normalization.pl --seqType fa --JM 60G  --max_cov 50 --CPU 12 --output /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/trinity_test2/insilico_read_normalization   --max_pct_stdev 10000  --left /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R1.fq --right /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R2.fq --pairs_together --PARALLEL_STATS  
> Converting input files. (both directions in parallel)CMD: ln -s /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R1.fq left.fa
> CMD: ln -s /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R2.fq right.fa
> CMD finished (0 seconds)
> CMD finished (0 seconds)
> CMD: touch left.fa.ok
> CMD finished (0 seconds)
> CMD: touch right.fa.ok
> CMD finished (0 seconds)
> Done converting input files.CMD: cat left.fa right.fa > both.fa
> CMD finished (5 seconds)
> CMD: touch both.fa.ok
> -------------------------------------------
> ----------- Jellyfish  --------------------
> -- (building a k-mer catalog from reads) --
> -------------------------------------------
> 
> CMD finished (0 seconds)
> CMD: jellyfish count -t 12 -m 25 -s 7858339627  --canonical  both.fa
> CMD finished (63 seconds)
> CMD: jellyfish histo -t 12 -o jellyfish.K25.min2.kmers.fa.histo mer_counts.jf
> CMD finished (4 seconds)
> CMD: jellyfish dump -L 2 mer_counts.jf > jellyfish.K25.min2.kmers.fa
> CMD finished (11 seconds)
> CMD: touch jellyfish.K25.min2.kmers.fa.success
> CMD finished (0 seconds)
> CMD: /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/..//Inchworm/bin/fastaToKmerCoverageStats --reads left.fa --kmers jellyfish.K25.min2.kmers.fa --kmer_size 25  --num_threads 12  --DS  > left.fa.K25.stats
> CMD: /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/..//Inchworm/bin/fastaToKmerCoverageStats --reads right.fa --kmers jellyfish.K25.min2.kmers.fa --kmer_size 25  --num_threads 12  --DS  > right.fa.K25.stats
> -reading Kmer occurrences...-reading Kmer occurrences...
> 
> 
>  done parsing 34555659 Kmers, 34555659 added, taking 55 seconds.
> 
>  done parsing 34555659 Kmers, 34555659 added, taking 55 seconds.
> STATS_GENERATION_TIME: 779 seconds.
> CMD finished (843 seconds)
> STATS_GENERATION_TIME: 789 seconds.
> CMD finished (852 seconds)
> CMD: touch left.fa.K25.stats.ok
> CMD finished (0 seconds)
> CMD: touch right.fa.K25.stats.ok
> -sorting each stats file by read name.
> CMD finished (0 seconds)
> CMD: /usr/bin/sort --parallel=12 -k5,5 -T . -S 30G left.fa.K25.stats > left.fa.K25.stats.sort
> CMD: /usr/bin/sort --parallel=12 -k5,5 -T . -S 30G right.fa.K25.stats > right.fa.K25.stats.sort
> CMD finished (0 seconds)
> CMD finished (1 seconds)
> CMD: touch left.fa.K25.stats.sort.ok
> CMD finished (0 seconds)
> CMD: touch right.fa.K25.stats.sort.ok
> CMD finished (0 seconds)
> CMD: /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/..//util/support_scripts//nbkc_merge_left_right_stats.pl --left left.fa.K25.stats.sort --right right.fa.K25.stats.sort --sorted > pairs.K25.stats
> -opening left.fa.K25.stats.sort
> -opening right.fa.K25.stats.sort
> -done opening files.
> **Error, pairs.K25.stats is empty.**  Be sure to check your fastq reads and ensure that the read names are identical except for the /1 or /2 designation. at /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/insilico_read_normalization.pl line 904.
> CMD finished (0 seconds)
> Error, cmd: /public-supool/home/niehu/anaconda2/opt/trinity-2.4.0/util/insilico_read_normalization.pl --seqType fa --JM 60G  --max_cov 50 --CPU 12 --output /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/trinity_test2/insilico_read_normalization   --max_pct_stdev 10000  --left /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R1.fq --right /public-supool/home/niehu/projects/others/PRJEB7345-mouse-embryo-RNAseq/05.denovo_assemble/trinity_denovo/test_R2.fq --pairs_together --PARALLEL_STATS   died with ret 512 at /public-supool/home/niehu/anaconda2/bin/Trinity line 2462.
> 

================================

├── chrysalis
├── insilico_read_normalization
│   └── tmp_normalized_reads
│       ├── both.fa
│       ├── both.fa.ok
│       ├── jellyfish.K25.min2.kmers.fa
│       ├── jellyfish.K25.min2.kmers.fa.histo
│       ├── jellyfish.K25.min2.kmers.fa.success
│       ├── left.fa
│       ├── left.fa.K25.stats
│       ├── left.fa.K25.stats.ok
│       ├── left.fa.K25.stats.sort
│       ├── left.fa.K25.stats.sort.ok
│       ├── left.fa.ok
│       ├── pairs.K25.stats
│       ├── right.fa
│       ├── right.fa.K25.stats
│       ├── right.fa.K25.stats.ok
│       ├── right.fa.K25.stats.sort
│       ├── right.fa.K25.stats.sort.ok
│       └── right.fa.ok
└── Trinity.timing

===========================================================
$ head insilico_read_normalization/tmp_normalized_reads/*
==> insilico_read_normalization/tmp_normalized_reads/both.fa <==
@HWUSI-EAS1726:1:100:10058:3078/1
CTCCCATCCAAGTACTAACCAGGCCCGACCCTGCTTAGCTTCCGAGATCAGACGAGATCGGGCGCGTTCA
+ERR632217.1001 HWUSI-EAS1726:1:100:10058:3078 length=76
GGEGGGGGGBGGEGGDGGGGGBGBGGGGGGGGGGBEDDCDGDGG<CEEEFEBGE<GEGG@B>CCCG@GDG
@HWUSI-EAS1726:1:100:10058:4350/1
CTCCCTCTCCGGAATCGAACCCTGATTCCCCGTCACCCGTGGTCACCATGGTAGGCACGGCGACTACCATCGAAAG
+ERR632217.1002 HWUSI-EAS1726:1:100:10058:4350 length=76
IIIIIIIIIIIIHIIIIIIIIIIIHIIIIIIIIHIIIIIIHEGIIIHHIIIEHIIGIIHIIGGGIGEFGDIGDFGE
@HWUSI-EAS1726:1:100:10058:6095/1
GGAATGCAGCCCAAAGCGGGTGGTAAACTCCATCTAAGGCTAAATACCGGCACGAGACCGATAGTCAACAAGTACC

==> insilico_read_normalization/tmp_normalized_reads/both.fa.ok <==

==> insilico_read_normalization/tmp_normalized_reads/jellyfish.K25.min2.kmers.fa <==
>100322
AAAAAAAAAAAAAAAAAAAAAAAAA
>4
CAGCTAAGGCTTATCCTTGAGTAGA
>4
AAAGCAGGCAGCACACTGAGGTCCA
>63
GACAAGGTGCCTTGGCTTGGCCCAC
>3
ACTTGTAGTCTTTGACTTCTCTGGA

==> insilico_read_normalization/tmp_normalized_reads/jellyfish.K25.min2.kmers.fa.histo <==
1 34042683
2 10698950
3 3529127
4 2647669
5 1872982
6 1519413
7 1231588
8 1043238
9 894758
10 780816

==> insilico_read_normalization/tmp_normalized_reads/jellyfish.K25.min2.kmers.fa.success <==

==> insilico_read_normalization/tmp_normalized_reads/left.fa <==
@HWUSI-EAS1726:1:100:10058:3078/1
CTCCCATCCAAGTACTAACCAGGCCCGACCCTGCTTAGCTTCCGAGATCAGACGAGATCGGGCGCGTTCA
+ERR632217.1001 HWUSI-EAS1726:1:100:10058:3078 length=76
GGEGGGGGGBGGEGGDGGGGGBGBGGGGGGGGGGBEDDCDGDGG<CEEEFEBGE<GEGG@B>CCCG@GDG
@HWUSI-EAS1726:1:100:10058:4350/1
CTCCCTCTCCGGAATCGAACCCTGATTCCCCGTCACCCGTGGTCACCATGGTAGGCACGGCGACTACCATCGAAAG
+ERR632217.1002 HWUSI-EAS1726:1:100:10058:4350 length=76
IIIIIIIIIIIIHIIIIIIIIIIIHIIIIIIIIHIIIIIIHEGIIIHHIIIEHIIGIIHIIGGGIGEFGDIGDFGE
@HWUSI-EAS1726:1:100:10058:6095/1
GGAATGCAGCCCAAAGCGGGTGGTAAACTCCATCTAAGGCTAAATACCGGCACGAGACCGATAGTCAACAAGTACC

==> insilico_read_normalization/tmp_normalized_reads/left.fa.K25.stats <==
1	59655.7	172353	288.912	4413;@;@;EDGGE<EE<E;B??BGDGGDBDDDD::?8*??:????<B??>	thread:1
1	66461.4	171685	258.323	12>0144431/1059@4=?6==@=>>B>>B9B>>;>;/;;3>77BB@@@##########################	thread:3
1	59188	164063	277.19	>1626B:88DDBBDEGG@GGG>GGDE>EEB?B?BBEB??=>;;8.325<-/+248::88?B??###	thread:2
1	67527.4	178644	264.551	<<<<:9:A><A/>5AA5AA.56);>A<AAA<A?6AAAAAAAAAA7719=42<42A??5A<A8:92A;??;;	thread:4
1	58256.3	164444	282.277	:8<>(12.41>/43:>55>0)-,<'1-81@@@@@@@9@@9===6@@@@@6<855=5675@@@@@	thread:4
1	58563	162677	277.781	;<===EEEEG=E@G@23@AGG@DD=DBD4EBB?E?*.5=768-);>;7'EE?B######################	thread:5
1	62196.3	169576	272.647	14424BB68;;;>6+676,;;;;87>BDBD<<>>318670;240;<B3B##########################	thread:3
1	65052.8	150468	231.301	:(80+B9=7BGBGDCGGGGGGGGGGG=DGCGGGBFEGGGGGGGGEGGGGGGGGGGGGDDGGGGGGBGGGGGGEGG	thread:3
1	63145.6	169126	267.834	2242223<>+@65<:@B@9>AC9C6268;5@553-400<607;826443;A/5?EE###################	thread:3
1	62381.9	172934	277.218	4+/(/;/<+A0=;;DBB>B-@@8;/.:;;DB@DDDB<>BBB:4;>>>><6BB?<><BB<6?>><9>BB#######	thread:5

==> insilico_read_normalization/tmp_normalized_reads/left.fa.K25.stats.ok <==

==> insilico_read_normalization/tmp_normalized_reads/left.fa.K25.stats.sort <==
1	55838.5	167029	299.129	###&43&;'14<43@@@<9@@6@@@5@@@AAAAAAAAAA@@@@@@@@@@@@@)@@@.@@################	thread:1
1	55374.8	158841	286.847	###+,#,*+25225@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@8@@@@@@@@@@	thread:1
1	9181	22818	248.535	###/.-*3.@@@@@@@@@@AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA	thread:1
1	56997.7	166438	292.008	###/<*7488()6<88>+<@@@9526,2=@@@@@@9@@@@@@@@<<<)6=<67<@@95@:0338@@59@@8@@##	thread:2
1	64055.6	175921	274.638	###0.06-0>=<=<,./2#-(/#06099/@@@5@65=6=@@@@533866@@@@995@@@86*07@99@9999@86	thread:5
1	60411	169688	280.889	###000220@AA@AAAA@@@@A@@A@@A@@@@@@@@@@@@@@@@@@@@@@8@#######################	thread:5
1	64203.3	176493	274.897	###033:88@@@B5@B@BBAAAAAA?<AAAAAAA@@B@@@@@@@AAAAA<<AAAAAAAA@@@6@@@@@@######	thread:3
1	63565.2	174004	273.74	###10#0#9-6-11:3<66@<<<<<<5@@@@@5@@9@@@@@@@@550;036869@@@9@################	thread:5
1	58494.4	163142	278.902	###10#1#91/)069?>5;<=?<=7<5=9@@@@@@<9@@68576@@@5@@@:5@#####################	thread:2
1	64595.3	180531	279.48	###3+')0''')/(/-+7>63659@@@@@995<7=<6=7@@@@@@@@@5@@@@@@@55@7=6<6@@@@@AAAAA1	thread:3

==> insilico_read_normalization/tmp_normalized_reads/left.fa.K25.stats.sort.ok <==

==> insilico_read_normalization/tmp_normalized_reads/left.fa.ok <==

==> insilico_read_normalization/tmp_normalized_reads/pairs.K25.stats <==

==> insilico_read_normalization/tmp_normalized_reads/right.fa <==
@HWUSI-EAS1726:1:100:10058:3078/2
TGAACGCGCCCGATCTCGTCTGATCTCGGAAGCTAAGCAGGGTCGGGCCTGGTTAGTACTTGGATGGGAG
+ERR632217.1001 HWUSI-EAS1726:1:100:10058:3078 length=76
IIIHIHGHIIIHGHHIHIDGIBGI>DIIIDHGEIHIFHGEGGEGGHDIGIGGEIEE8CACGGGCEGGE2G
@HWUSI-EAS1726:1:100:10058:4350/2
CGACGACCCATTCGAACGTCTGCCCTATCAACTTTCGATGGTAGTCGCCGTGCCTACCATGGTGACCACGGGTGAC
+ERR632217.1002 HWUSI-EAS1726:1:100:10058:4350 length=76
IIIIIIIIIIIIIIIIIIIHIIIIIIIHHIIIIIIIIHHIIIIIIIIIIIIHHIIIIDHIHHGIHIIHIIFIBEDG
@HWUSI-EAS1726:1:100:10058:6095/2
GAACTCTCTCTTCAAAGTTCTTTTCAACTTTCCCTTACGGTACTTGTTGACTATCGGTCTCGTGCCGGTATTTAGC

==> insilico_read_normalization/tmp_normalized_reads/right.fa.K25.stats <==
1	67333.8	190154	282.404	22/.6@@54;5@2920<33B?<BB;=A;=/==5<?CC@?99=<8A>@2?+54813573011047>A@?>DB=B=A	thread:0
1	61355.8	162391	264.671	2>-4/1112;A9@;@5@9/7>/-30>/>1/115811111A=;A985125+8337??67?5<66;;AA;?######	thread:5
1	64521.4	165179	256.006	>02.-91/.B:<B??A?>9A>=>?9??:???>=<DDD<3@C?>C@DBBDGCG@DEE@EGA?<AACCBBB@A??@>	thread:3
1	65816.4	177291	269.372	42>0*6;1<ECCBC??B?????8>?BC??DDDG@?8>=387.:5AA?>?EBBC>?@;ECBB@C>3=?=?@@@>@8	thread:4
1	60724.4	168343	277.224	/..40.42*746/;7>1/<FBFA<>C>8A6513.?796<5=-=<:?975B@<B##################	thread:3
1	57299.2	163540	285.414	420.219<7ABBB?DGGGBF>BAC?B?B@CCCCAEEGEGGEEFEIHIBECEEEFDHHDECECDCBBB=BBBB	thread:3
1	59882.4	165920	277.076	2<0,8@7:@<C=8>06445E<D8B64?334/43-889@5.35-<3531567;<9AE>AEGDBGD:??=?==;;A9	thread:1
1	63530.1	172316	271.235	>722.26;7DG<DGID@EIGBB@<8<;92-7967GDBD<=<8=<)97<<-3156	thread:5
1	61927.7	168086	271.423	+252/4?1(:<7@77==7?;>?C>DGGG><??@?>BAEADGBG<ADD3AA;A??CAAAC@@DD>;?<A?8????E	thread:2
1	61009.8	162593	266.503	42/)44>41GGDGDBGGB<C>CC>9:=?,9958<56669ACAAC??<A?-1-4:@?2A?<4136(:,35@@@=22	thread:1

==> insilico_read_normalization/tmp_normalized_reads/right.fa.K25.stats.ok <==

==> insilico_read_normalization/tmp_normalized_reads/right.fa.K25.stats.sort <==
1	61010.1	172078	282.049	'''''/''>***(,--+(:3111+98<==DG@DDIIHIIIIIIHBHIIFIG	thread:4
1	59607	169086	283.669	'''*;*),'8.+/1).<)+''*118;A;@EEFFCDGGEGEGGGGGGEE	thread:2
1	64087.1	182406	284.623	'''/6*081>C;C,486367+529BDBBDGAEG>FE<FEGDGGGDGE@DGGBGDFEE<E	thread:1
1	67464.2	180811	268.01	'''2''*04B?@?BCC228AA8ACBD@GGDGGGGID<I@IIIGIGBG8GGGGEBEGGDFIHGFEFGFCDDEB38@	thread:5
1	56396.8	167966	297.83	''/('-/.':-?2;=2=1=*>::8@?<<BGGDGFC	thread:3
1	86417.8	239633	277.296	''>21-?43GGBGDBDDFDE28E=8=A@@BDEFFHIIIGIIIGHIII<IIGIIIB=IIIDEBDEIFIIF	thread:4
1	53361	161549	302.747	'('2),6<5(.42<A1;94F3F8>@FF@@IHIDHGIGIIHIIIIIHHHIBGGGGGFII	thread:0
1	55654.1	170672	306.665	'('8..24(7BA?57B27=-.94.D8DDBGEGIIGFDEIFGIHIIFIIIIFHIDIF><IE?CF?8CCC>@=A??C	thread:2
1	55458.6	163973	295.668	'(*(1(13.AE<EG>GDDA2<:9;CFCE@GGEGGIIIIIIIEIIIGIIFIIGGGGIII	thread:1
1	62138	174399	280.664	'(*6+))0701=22+(.1<-22;4<7<B<GDGDGGGGBG@BDEE>B7>>A<=>>;9@	thread:3

==> insilico_read_normalization/tmp_normalized_reads/right.fa.K25.stats.sort.ok <==

==> insilico_read_normalization/tmp_normalized_reads/right.fa.ok <==

###################################3
The K25.stats  file are very strange. 

Best,
Hu

**Comment by fynfyn** [comment]
I have experienced the problem before. The problem is your fasta/fastq file have not the correct type. you can try to use some command to make it right.  You can try this
zcat your_file_name_1.gz | awk '{if (NR%4==1) {print $1'_'$2/1'} else {prnt} }' gzip  > your_file_name_1_new.gz     
or you can try this 
 zcat your_file_name_2.gz | awk '{if (NR%4==1) {print $1'_'$2/2'} else {prnt} }' gzip  > your_file_name_2_new.gz

**Comment by niehu2018** [comment]
read 1 id:
@HWUSI-EAS1726:1:100:10059:10049/1
read 2 id:
@HWUSI-EAS1726:1:100:10059:10049/2

there is no $2 in id line.

I tried to rename the id, but get the same result.

**Comment by niehu2018** [comment]
I followed your advice. @manuelsmendoza , it still did not work.
I found my problem, it's stupid. Sorry to not paste my cmd line into this issue!
In the cmd line, I specified --seqType fa, and Trinity does not try to compare what I specified with the input file. As a result, the quality scores were written into K25.stats file.
I changed "--seqType fa" to "--seqType fq", and everything is ok.
Thanks! @manuelsmendoza @fynfyn .
